docs/en/observability/apm/view-and-analyze/inventory.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[inventory]]
+= Inventory
+
+++++
+<titleabbrev>Inventory</titleabbrev>
+++++
+
+Inventory provides a single place to observe the status of your entire ecosystem of hosts, containers, and services at a glance, even just from logs. From there, you can monitor and understand the health of your entities, check what needs attention, and start your investigations.
+
+NOTE: The new Inventory requires the Elastic Entity Model (EEM). To learn more, refer to <<elastic-entity-model>>.
+
+[role="screenshot"]
+image::./images/inventory-catalog.png[Inventory catalog]
```

Comment:
This screenshot shows the serverless navigation (plus it's stale). You should crop the screenshot so that it does not include the nav pane. (In general, I'd avoid including the nav pane because there will be more than one version of the nav in 8.16.

Reply:
Good point! Thanks, @dedemorton!